lib/gatling/init_script_template.sh.eex

```diff
@@ -13,6 +13,6 @@ export MIX_ENV=prod
 export PORT=<%= @port %>
 export HOME=<%= System.user_home %>/deployments/<%= @project_name %>
 
-export $(cat <%= System.user_home %>/deployments/<%= @project_name %>/.env)
+[ -f $HOME/.env ] && export $(cat $HOME/.env)
```

Comment:
I think the path was correct as before, see https://github.com/hashrocket/gatling/blob/master/lib/gatling/tasks/config/set.ex#L16